lib/controllers/edificio_controller.test.js

```diff
@@ -0,0 +1,64 @@
+import request from 'supertest';
+import { cleanDb } from '../../test/db_utils';
+import app from '../app';
+import Edificio from '../models/edificio';
+
+describe('Edificio controller', () => {
+  beforeAll(async () => {
+    await cleanDb();
+
+    await Edificio.bulkCreate([
+      { nombre: 'Malvinas' },
+      { nombre: 'Origone A' },
+    ]);
+  });
+
+  describe('/edificios', () => {
+    it('devuelve código 200', async () => {
+      const response = await request(app).get('/api/edificios');
+      expect(response.statusCode).toBe(200);
+    });
+
+    it('devuelve la lista de edificios', async () => {
+      const response = await request(app).get('/api/edificios');
+      expect(response.body).toMatchObject({
+        data: [{ nombre: 'Malvinas' }, { nombre: 'Origone A' }],
+      });
+    });
+  });
+
+  describe('/edificios/id', () => {
+    it('devuelve código 200', async () => {
+      const response = await request(app).get('/api/edificios/1');
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toMatchObject({
+        data: { nombre: 'Malvinas' },
+      });
+    });
+
+    it('devuelve código 404', async () => {
```

Comment:
Estaría bueno meter un `describe` que explique cuál es el caso que se está testeando. Algo así:

```suggestion
    describe('cuando el edificio no existe', () => {
      it('devuelve código 404', async () => {
```